frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx

```diff
@@ -109,6 +109,18 @@ const DiskReplacementAction = (props: DiskReplacementActionProps) => {
       <ModalTitle>Disk Replacement</ModalTitle>
       <ModalBody>
         <p>This action will start preparing the disk for replacement.</p>
+        {isRebalancing && alertsMap[diskName].status !== Status.Offline && (
+          <Alert
+            variant="info"
+            className="co-alert"
+            title="Data rebalancing is in progress"
+            isInline
+          >
+            <Link onClick={close} to="/dashboards/persistent-storage">
```

Comment:
can you create constant for 'persistent-storage' and use it in here & in dashboard name so we don't forget to change once dashboards are shifted.

Reply:
I am strongly against making constants out of things which are only used in one place. We should be wary of changing these minor details when moving the dashboard without the need of making constants.

Reply:
We are making the change because this is used in more than one place.